zeebe/engine/src/main/java/io/camunda/zeebe/engine/state/user/UserRecordValue.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.zeebe.engine.state.user;
+
+import io.camunda.zeebe.db.DbValue;
+import io.camunda.zeebe.msgpack.UnpackedObject;
+import io.camunda.zeebe.msgpack.property.ObjectProperty;
+import io.camunda.zeebe.protocol.impl.record.value.user.UserRecord;
+
+public class UserRecordValue extends UnpackedObject implements DbValue {
```

Comment:
❌ This naming is confusing. `RecordValue` is what we call our records in the protocol. Let's avoid using the same naming pattern here. I would opt to rename it to `StoredUser` or `PersistedUser`. This in line with `StoredMessage` or `PersistedProcess` respectively.